MaterialLibrary/src/main/java/io/doist/material/widget/FloatingActionButton.java

```diff
@@ -91,22 +85,8 @@ private void init(Context context, AttributeSet attrs, int defStyleAttr) {
                             elevation);
                 }
 
-                padding = ta.getDimensionPixelSize(R.styleable.FloatingActionButton_android_padding, padding);
-                paddingLeft = ta.getDimensionPixelSize(R.styleable.FloatingActionButton_android_paddingLeft, padding);
-                paddingTop = ta.getDimensionPixelSize(R.styleable.FloatingActionButton_android_paddingTop, padding);
-                paddingRight = ta.getDimensionPixelSize(R.styleable.FloatingActionButton_android_paddingRight, padding);
-                paddingBottom = ta.getDimensionPixelSize(R.styleable.FloatingActionButton_android_paddingBottom, padding);
-
                 // Resolve color or colorStateList.
-                TypedValue v = new TypedValue();
-                if (ta.getValue(R.styleable.FloatingActionButton_android_color, v)) {
-                    if (v.type >= TypedValue.TYPE_FIRST_COLOR_INT
-                            && v.type <= TypedValue.TYPE_LAST_COLOR_INT) {
-                        color = ColorStateList.valueOf(v.data);
-                    } else {
-                        color = ta.getColorStateList(R.styleable.FloatingActionButton_android_color);
-                    }
-                }
+                color = ta.getColorStateList(R.styleable.FloatingActionButton_android_color);
```

Comment:
I thought this was going to break providing a raw color (`#abc`), but it works! :smile: :+1: